lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -408,8 +408,13 @@ module InternalConstants
 
     # Whether valid entries should be written even if some other entries fail
     # due to INVALID_ARGUMENT or PERMISSION_DENIED errors when communicating to
-    # the Stackdriver Logging API. This is highly recommended.
-    config_param :partial_success, :bool, :default => true
+    # the Stackdriver Logging API. This flag is no longer used, and is kept for
+    # backwards compatibility, partial_success is enabled for all requests.
+    # TODO: Breaking change. Remove this flag in Logging Agent 2.0.0 release.
+    config_param :partial_success, :bool,
+                 :default => true,
+                 :skip_accessor => true,
+                 :deprecated => 'This feature is permanently enabled by default'
```

Comment:
Nit: "by default" implies that it could be disabled. I'd just say `'This feature is permanently enabled'`.

Reply:
Done.